app/serializers/api/current_order_serializer.rb

```diff
@@ -15,4 +15,9 @@ def payment_method_id
   def display_total
     object.display_total.money.to_f
   end
+
+  def shipping_method_id
+    return unless object.shipments.any?
+    object.shipments.last.shipping_method_id
+  end
```

Comment:
Maybe we can add some documentation here explaining why we need this and maybe telling that in the future we should move from `order.shipments.last` to something more robust